ProcessLib/LIE/HydroMechanics/LocalAssembler/HydroMechanicsLocalAssemblerMatrixNearFracture.h

```diff
@@ -72,6 +76,11 @@ class HydroMechanicsLocalAssemblerMatrixNearFracture
 
     static const int displacement_jump_index =
         displacement_index + displacement_size;
+
+    std::vector<FractureProperty*> _fracture_props;
```

Comment:
Might be good to use std::vector<std::reference_wrapper<const unique_ptr<FractureProperty>> > because the vector is pushed with process_data.fracture_property.get().

Reply:
thanks for the suggestion. however that way is too complicated for me